demos/device-shadow-for-aws-iot-embedded-sdk/shadow_demo_helpers.c

```diff
@@ -0,0 +1,916 @@
+ * must be unique so edit as required to ensure no two clients connecting to the
+ * same broker use the same client identifier.
+ */
+#ifndef democonfigCLIENT_IDENTIFIER
+    #define democonfigCLIENT_IDENTIFIER    "testClient"__TIME__
+#endif
+
+/**
+ * @brief The port to use for the demo.
+ */
+#ifndef democonfigMQTT_BROKER_PORT
+    #define democonfigMQTT_BROKER_PORT    ( 1883 )
+#endif
+
+/**
+ * @brief The maximum number of times to run the subscribe publish loop in this
+ * demo.
+ */
+#ifndef democonfigMQTT_MAX_DEMO_COUNT
+    #define democonfigMQTT_MAX_DEMO_COUNT    ( 3 )
+#endif
+
+/*-----------------------------------------------------------*/
+
+/**
+ * @brief Timeout for receiving CONNACK packet in milliseconds.
+ */
+#define mqttexampleCONNACK_RECV_TIMEOUT_MS           ( 1000U )
+
+/**
+ * @brief The topic to subscribe and publish to in the example.
+ *
+ * The topic name starts with the client identifier to ensure that each demo
+ * interacts with a unique topic name.
+ */
+#define mqttexampleTOPIC                             democonfigCLIENT_IDENTIFIER "/example/topic"
+
+/**
+ * @brief The number of topic filters to subscribe.
+ */
+#define mqttexampleTOPIC_COUNT                       ( 1 )
+
+/**
+ * @brief The MQTT message published in this example.
+ */
+#define mqttexampleMESSAGE                           "Hello World!"
+
+/**
+ * @brief Dimensions a file scope buffer currently used to send and receive MQTT data
+ * from a socket.
+ */
+#define mqttexampleSHARED_BUFFER_SIZE                ( 500U )
+
+/**
+ * @brief Time to wait between each cycle of the demo implemented by prvMQTTDemoTask().
+ */
+#define mqttexampleDELAY_BETWEEN_DEMO_ITERATIONS     ( pdMS_TO_TICKS( 5000U ) )
+
+/**
+ * @brief Timeout for MQTT_ProcessLoop in milliseconds.
+ */
+#define mqttexamplePROCESS_LOOP_TIMEOUT_MS           ( 500U )
+
+/**
+ * @brief Keep alive time reported to the broker while establishing an MQTT connection.
+ *
+ * It is the responsibility of the Client to ensure that the interval between
+ * Control Packets being sent does not exceed the this Keep Alive value. In the
+ * absence of sending any other Control Packets, the Client MUST send a
+ * PINGREQ Packet.
+ */
+#define mqttexampleKEEP_ALIVE_TIMEOUT_SECONDS        ( 60U )
+
+/**
+ * @brief Delay between MQTT publishes. Note that the process loop also has a
+ * timeout, so the total time between publishes is the sum of the two delays.
+ */
+#define mqttexampleDELAY_BETWEEN_PUBLISHES           ( pdMS_TO_TICKS( 500U ) )
+
+/**
+ * @brief Transport timeout in milliseconds for transport send and receive.
+ */
+#define mqttexampleTRANSPORT_SEND_RECV_TIMEOUT_MS    ( 200U )
+
+/**
+ * @brief Maximum number of outgoing publishes maintained in the application
+ * until an ack is received from the broker.
+ */
+#define MAX_OUTGOING_PUBLISHES                       ( 1U )
+
+/**
+ * @brief Milliseconds per second.
+ */
+#define _MILLISECONDS_PER_SECOND                     ( 1000U )
```

Comment:
Please help and remove the leading underscore from macros.
```suggestion
MILLISECONDS_PER_SECOND
```

Reply:
will update them.